source/ed/connectors/fusio_parser.cpp

```diff
@@ -623,10 +628,19 @@ std::vector<ed::types::VehicleJourney*> TripsFusioHandler::get_split_vj(Data& da
         }
         vj->uri = vj_uri;
 
-        if (is_valid(headsign_c, row))
+        if (is_valid(headsign_c, row)) {
+            vj->headsign = row[headsign_c];
```

Comment:
```suggestion
            vj->headsign = row[trip_headsign_c];
```
Same as for the stop time. It would ease readability soooooo much to have proper naming for the column :pray: